### DISCUSSION TITLE
如何隐藏序号列

### DISCUSSION THREAD

**Discussion by lmd0955** [dream-num/Luckysheet#932] [Q&A]
![image](https://user-images.githubusercontent.com/45345751/149885680-7a03e9dc-8734-46e2-bf85-aa295751c00c.png)
用来做布局，设计好以后需要把行列序号隐藏，该怎么操作

**Comment by hzmings** [comment]
rowHeaderWidth: 0,				//行标题区域的宽度
columnHeaderHeight: 0,			//列标题区域的高度
设置为0试试

**Comment by woxiaolf** [comment]
这两个值 如何通过两个按钮去控制？
